.pre-commit-config.yaml

```diff
@@ -23,6 +23,6 @@ repos:
 - repo: https://github.com/rhysd/actionlint
   rev: v1.6.23
   hooks:
-  - id: actionlint
+  - id: actionlint-docker
```

Comment:
I tested it locally, shellcheck thinks the square brackets in `pip install haystack[all]` need expansion. I think we can safely ignore the error by adding:
```yaml
    args: ["-ignore", "SC2102"]
```